src/webgpu/api/operation/buffers/map_oom.spec.ts

```diff
@@ -99,14 +99,15 @@ an out-of-memory error if allocation fails.
 
     let mapping: ArrayBuffer | undefined = undefined;
     if (oom) {
-      // Note: It is always valid to get mapped ranges of a GPUBuffer that is mapped at creation, even if it is invalid,
-      // because the Content timeline might not know it is invalid.
-      t.shouldThrow('OperationError', f);
+      // getMappedRange is normally valid on OOM buffers, but this one fails because the
+      // (default) range is too large to create the returned ArrayBuffer.
+      t.shouldThrow('RangeError', f);
```

Comment:
Is it because this should be a `RangeError`? (spec now says `OperationError` but that should be changed)

Reply:
I believe this getMappedRange call is not actually hitting any of the validation cases that result in OperationError in the spec. Instead, it's failing inside:
> Let m be a new ArrayBuffer of size rangeSize

which can fail with a RangeError if the size is too large.

Reply:
e.g.: `new ArrayBuffer(0x2000000000)`

```
VM67:1 Uncaught RangeError: Array buffer allocation failed
    at new ArrayBuffer (<anonymous>)
    at <anonymous>:1:1
```